lib/fruitstrap.js

```diff
@@ -0,0 +1,52 @@
+import { exec } from 'teen_process';
+import logger from './logger';
+
+const FRUITSAP_PATH = 'fruitstrap';
+
+class Fruitstrap {
+ 
+  constructor (udid, opts = {}) {
+    this.udid = udid;
+    this.fruitstrapPath = opts.path || FRUITSAP_PATH;
+  }
+
+  async remove (bundleid) {
+    let remove = [`uninstall`, `--id`, `${this.udid}`, `--bundle`, `${bundleid}`];
```

Comment:
I think naming this variable `args` (or perhaps `removeArgs`) makes it more clear what is going on. It's not a big point, but it is how the rest of Appium is generally structured.